policies/optimized-images.md

```diff
@@ -239,104 +138,94 @@ For an `<img>` element, if neither the width or the height of the source image e
 
-- The default allowlist for `image-compression` is `*`. This means for pages of all origins, `<img>` elements whose file sizes exceeds the compression ratio will be allowed and rendered correctly.
+- The default allowlist for `unoptimized-images` is `*`. This means for pages of all origins, `<img>` elements whose file sizes exceeds the compression ratio will be allowed and rendered correctly.
 
 
-- A `image-compression` policy can be specified via:
+- A `unoptimized-images` policy can be specified via:
 
     **1. HTTP "feature-policy" response header:**
     ```html
-    Feature-Policy: image-compression 'none';
+    Feature-Policy: unoptimized-images 'none';
     ```
-    In this example, `image-compression` is disabled for all frames including the main frame. All `<img>` elements whose file sizes exceeds the compression ratio will be rendered with inverted colors.
+    In this example, `unoptimized-images` is disabled for all frames including the main frame. All `<img>` elements whose file sizes exceeds the limit will be rendered as placeholder images.
 
     **2. "allow" attribute in <iframe>:**
     ```html
-    <iframe src="https://example.com" allow="image-compression 'self' https://foo.com;">
+    <iframe src="https://example.com" allow="unoptimized-images 'self' https://foo.com;">
     ```
-    In this example, `image-compression` is disabled everywhere except on the origin of the main document and on `https://foo.com`.
+    In this example, `unoptimized-images` is disabled everywhere except on the origin of the main document and on `https://foo.com`.
 
 
 #### Examples
 
 <table>
   <tr align="center">
-   <td width="400">Feature-Policy: image-compression 'none'; </td>
-   <td width="400">Feature-Policy: image-compression *; </td>
+   <td width="400">Feature-Policy: unoptimized-images 'none'; </td>
+   <td width="400">Feature-Policy: unoptimized-images *; </td>
   </tr>
   <tr align="center">
    <td>
- <img src="resources/image-compression-disabled.png" width="80%"> 
+ <img src="resources/unoptimized-disabled.png" width="80%"> 
    </td>
    <td>
- <img src="resources/image-compression-enabled.png" width="80%"> 
+ <img src="resources/unoptimized-enabled.png" width="80%"> 
    </td>
   </tr>
-  <tr align="center">
-   <td colspan="2" >
+</table>
+
+For an `<img>` element, if its file size is within the limit, the image will be rendered correctly; otherwise the image will be rendered as placeholder images.
 
-```html
-"example.com"
-<img id="normal-size" src="test.png">
-<img id="oversized" src="test-oversized.png">
+
+**Future Development**
+
+Image formats affect file size. We want to support different default values for different image formats.
+We want to allow developers to specify the parameters as well. In practice, they would look something like this:
+```html    
+<iframe allow="unoptimized-images(BMP(1, 0.5), JPG(2, 0.2))"></iframe>
```

Comment:
We should probably use the MIME types of image/bmp and image/jpeg (or just 'bmp' or 'jpeg') here instead of having to define a new mapping of strings to image formats.

Reply:
OK. But the MIME types are sometimes img/jpg image/jpg and stuff so it is hard to include all here. I guess it'd make it challenging to implement too.